app/src/main/java/com/framgia/bitcoinwallet/data/model/BaseTransaction.kt

```diff
@@ -1,7 +1,7 @@
 package com.framgia.bitcoinwallet.data.model
 
 open class BaseTransaction {
-    protected var amount: Float = 0.0f
-    protected lateinit var note: String
-    protected var timestamp: Long = 0
-}
\ No newline at end of file
+    var amount: Float = 0.0f
+    lateinit var note: String
```

Comment:
anh chưa test thử khi mà server ko trả về cái này thì nó sẽ ra sao
nhưng anh nghĩ tốt nhất là nên để var hoặc val
not null thì nên có default value còn nullable thì default null

Reply:
Thế em cứ để var với default value a nhỉ